examples/website/geospatial/app.tsx

```diff
@@ -123,6 +123,8 @@ export default function App(props: AppProps) {
   const [state, setState] = useState<AppState>({
     table: null,
     viewState: INITIAL_VIEW_STATE,
+    selectedCategory: null,
```

Comment:
Better save `example` here rather than the names.

Reply:
Done